app/upgrades.go

```diff
@@ -58,3 +73,69 @@ func (app *Quicksilver) setUpgradeStoreLoaders() {
 		app.SetStoreLoader(upgradetypes.UpgradeStoreLoader(upgradeInfo.Height, storeUpgrades))
 	}
 }
+
+// TODO: Rename.
+const UpgradeName = "v046-to-v047"
```

Comment:
We should try and keep this inline with how we handle upgrades.

We have also removed crisis in v0.47 - so this should be mirrored here too.